### PR TITLE
cartservice - dotnet 6.0.2

### PR DESCRIPTION
**Pull request by mathieu-benoit**
.NET Core just got a new version: [6.0.2](https://devblogs.microsoft.com/dotnet/february-2022-updates/)

### CODE REVIEW

**Comment by mathieu-benoit** [comment]
Ready for your review, thanks!

File: src/cartservice/src/cartservice.csproj [hunk 1]
```diff
@@ -5,8 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.AspNetCore" Version="2.41.0" />
-    <PackageReference Include="Grpc.HealthCheck" Version="2.42.0" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.42.0" />
+    <PackageReference Include="Grpc.HealthCheck" Version="2.43.0" />
```
[the remark this answers is not in the document]
Comment (mathieu-benoit) [reply src/cartservice/src/cartservice.csproj:9]: Good question, I actually have a dependabot setup on my own [`cartservice`](https://github.com/mathieu-benoit/cartservice/blob/main/.github/dependabot.yml), and then once notified I sometimes port them here too ;) Maybe we could check if/why the renovate/dependabot is not tackling them here? Let me know if I could help.

[the remark this answers is not in the document]
Comment (mathieu-benoit) [reply src/cartservice/src/cartservice.csproj:9]: Oh actually it seems they are tracked in there https://github.com/GoogleCloudPlatform/microservices-demo/pull/700

[the remark this answers is not in the document]
Comment (NimJay) [reply src/cartservice/src/cartservice.csproj:9]: Ahh, got it, thank you! Good to know that https://github.com/mathieu-benoit/cartservice is catching things for us while we wait for #700. 😅

**Pull request merged**